process/block/preprocess/transactions.go

```diff
@@ -451,6 +446,10 @@ func (txs *transactions) CreateAndProcessMiniBlock(sndShardId, dstShardId uint32
 		}
 	}
 
+	if len(miniBlock.TxHashes) < 3 && len(miniBlock.TxHashes) > 0 {
```

Comment:
delete bubu

Reply:
I can not believe this got through! Surely will be deleted!

Reply:
Deleted by @SebastianMarian